Resources/Prototypes/_Mono/explosion.yml

```diff
@@ -38,10 +38,10 @@
   id: HardBombShipGun
   damagePerIntensity:
     types:
-      Heat: 7.5
-      Blunt: 7.5
-      Piercing: 3
-      Structural: 20
+      Heat: 2
+      Blunt: 2
+      Piercing: 0.5
+      Structural: 2
```

Comment:
this is below what normal explosions do
should just tweak explosion parameters and leave those as high-structural low-normal-type explosion damage specs

Reply:
should probably be changed to do big structural and as-is other types